dom.bs

```diff
@@ -7294,14 +7310,26 @@ method steps are:
  <li><p>Let <var>attribute</var> be the first <a>attribute</a> in <a>this</a>'s
  <a for=Element>attribute list</a> whose <a for=Attr>qualified name</a> is <var>qualifiedName</var>,
  and null otherwise.
- <!-- This is step 2 of "get an attribute by name", modified as appropriate -->
+
+ <li><p>If <var>attribute</var> is null, then set <var>attribute</var> to an <a>attribute</a>
+ whose <a for=Attr>local name</a> is <var>qualifiedName</var>, <a for=Attr>value</a> is
+ <var>value</var>, and <a for=Node>node document</a> is <a>this</a>'s <a for=Node>node document</a>.
```

Comment:
Is the idea here to create a dummy attribute that you don't append in order to calculate the verified value?

Why is that done this way? This is quite confusing to read through.

Reply:
Apologies for the slow reply on here I thought I replied before but it must have got lost.

TLDR: I agree with you I can't really remember why it's done this way. I updated the TT spec to just take the local name and namespace directly rather than an attribute object. As part of doing this change I realised these algorithms were overly complicated for what was actually needed.

Hopefully now the integration points should be much cleaner and less invasive.